tests/pmap_test.py

```diff
@@ -1005,7 +1008,7 @@ def testReshardInput(self):
     # Manually construct a ShardedDeviceArray with the wrong sharding for the
     # subsequent pmap
     shard_shape = (3,2)
-    shard = jnp.arange(jnp.prod(shard_shape)).reshape(shard_shape)
+    shard = jnp.arange(jnp.prod(jnp.array(shard_shape))).reshape(shard_shape)
```

Comment:
Out of curiosity, what warning does this squash?

Reply:
This'n: https://github.com/google/jax/blob/ebc5e8bfd6d091bf1fbf67de1565705f30159aaf/jax/numpy/lax_numpy.py#L1652-L1655